tcpMaxConn.go

```diff
@@ -1,24 +1,25 @@
 package main
 
 import (
-	"bufio"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
-	"strconv"
-	"strings"
-
-	"github.com/dachad/check-max-tcp-connections/mtcpclient"
 	"github.com/spf13/pflag"
+	"github.com/dachad/check-max-tcp-connections/mtcpclient"
+	"github.com/dachad/check-max-tcp-connections/cmdutil"
+	"github.com/dachad/check-max-tcp-connections/tcpclient"
 )
 
+
+
 func main() {
 	hostPtr := pflag.StringP("host", "h", "", "Host you want to open tcp connections against (Required)")
 	// according to https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers, you are probably not using this
```

Comment:
get rid of this comment, not longer used

Reply:
good point!